src/gui/mainwindow.h

```diff
@@ -81,6 +81,12 @@ class MainWindow: public QMainWindow, private Ui::MainWindow
     QMenu* getTrayIconMenu();
     PropertiesWidget *getProperties() const { return properties; }
 
+    // ExecutionLog properties
+    bool isExecutionLogEnabled() const;
+    void setExecutionLogEnabled(bool value);
+    int getExecutionLogMsgTypes() const;
```

Comment:
Why do you use non-Qt naming? Just `executionLogMsgTypes` as we usually did. This affects all this PR.